data/strings/types_strings.txt

```diff
@@ -3852,8 +3852,10 @@
     zh-Hant = 淋浴間
 
   [type.amenity.taxi]
```

Comment:
If we already touched it, can you please later update all translations for the taxi (and in categories.txt) in a separate PR?
```
en:Taxi Stand
ar:محطة التاكسي
be:Стаянка таксі
bg:Стойка за таксита
ca:Parada de taxis
cs:Stanoviště taxi
da:Taxaholdeplads
de:Taxistand
el:Περίπτερο ταξί
es:Parada de taxis
es-MX:Parada de taxi
et:Taksopeatus
eu:Taxi geltokia
fa:ایستگاه تاکسی
fi:Taksitolppa
fr:Station de taxis
he:תחנת מוניות
hu:Taxi stand
id:Tempat taksi
it:Stazione dei taxi
ja:タクシースタンド
ko:택시 승차장
lt:Taksi stendas
mr:टॅक्सी स्टँड
nb:Drosjeholdeplass
nl:Taxistandplaats
pl:Postój taksówek
pt:Ponto de táxi
ro:Stație de taxi
ru:Стоянка такси
sk:Stanovište taxíkov
sv:Taxistation
sw:Stendi ya teksi
th:ป้ายรอแท็กซี่
tr:Taksi durağı
uk:Стоянка таксі
vi:Đón taxi
zh-Hans:出租车站
zh-Hant:的士站
```

Reply:
@vng will taxi be searchable by a single `taxi` word in all these cases above?

Reply:
Ok, will do that. 
From a quick test it looks like `taxi` should probably be searchable in other languages as well if one uses: `en:3Taxi|Taxi Stand`

Reply:
I struggled to build the project with QT6 and gave up for now, so I won't be able to change the taxi translations and test the search in time for the next release.

Reply:
@map-per what kind of build errors did you get?

Reply:
I'm getting this error:
```
map-per@Linux-PC:~/Dokumente/organicmaps$ tools/unix/build_omim.sh -r desktop
-- Using compiler GNU 11.3.0
-- Using Unity Build with batch 50, export UNITY_DISABLE=1 or use -DUNITY_DISABLE=ON to disable it.
-- export COLORS_DISABLE=1 or use -DCOLORS_DISABLE=ON to disable colored compiler output.
Setting PLATFORM_LINUX to true
-- Build type: Release
-- Using ld.gold
CMake Error at /usr/lib/x86_64-linux-gnu/cmake/Qt6/QtFeature.cmake:1212 (message):
  Feature rpath is already defined to be "ON" and should now be set to "OFF"
  when importing features from Qt6::Core.
Call Stack (most recent call first):
  /usr/lib/x86_64-linux-gnu/cmake/Qt6Core/Qt6CoreConfig.cmake:128 (qt_make_features_available)
  /usr/lib/x86_64-linux-gnu/cmake/Qt6/Qt6Config.cmake:219 (find_package)
  CMakeLists.txt:244 (find_package)


-- Configuring incomplete, errors occurred!
See also "/home/map-per/Dokumente/omim-build-release/CMakeFiles/CMakeOutput.log".
```

Maybe I messed up something while trying to install Qt6. Should the Installation work with a clean Ubuntu22 Installation? If so I should maybe reset the computer.